pkg/sbproxy/reconcile/reconcile_visibilities.go

```diff
@@ -63,7 +63,7 @@ func (r *resyncJob) brokerNames(brokers []*platform.ServiceBroker) []string {
 	return names
 }
 
-func (r *resyncJob) getSMBrokerPlans(ctx context.Context, offerings map[string]*types.ServiceOffering, smBrokers []*platform.ServiceBroker) (map[string]brokerPlan, error) {
+func (r *resyncJob) getSMBrokerPlans(ctx context.Context, offerings map[string]*types.ServiceOffering, smBrokers []*platform.ServiceBroker) (map[string]map[string]brokerPlan, error) {
 	log.C(ctx).Info("resyncJob getting service plans from Service Manager")
```

Comment:
maybe create type for map[string]map[string]brokerPlan -> plansByBroker

Reply:
resolving the comment because it is not relevant after the impl refactor